internal/manifest/coi_iso_tree.go

```diff
@@ -66,6 +67,24 @@ func (p *CoreOSISOTree) serialize() osbuild.Pipeline {
 		osbuild.NewFilesInputs(osbuild.NewFilesInputReferencesPipeline(p.payloadPipeline.Name(), p.payloadPipeline.Filename)),
 	))
 
+	if p.coiPipeline.Ignition != nil {
+		cso := osbuild.CopyStageOptions{}
+		filename := ""
+		copyInput := ""
+		if p.coiPipeline.Ignition.Data64 != "" {
+			filename = "ignition_config"
```

Comment:
What reads this filename?

Reply:
It is passed to a format string in the `CopyStageOptions` a couple of lines below

Reply:
Ahh OK, I hadn't realized the dracut glue picked it up by default.  This seems very much worth adding as a comment into the code.

Reply:
Will do :+1: